pyproject.toml

```diff
@@ -40,7 +40,7 @@ nilearn = "^0.6"
 colorlog = "^5"
 xgboost = "*"
 scipy = "^1.2.3"
-matplotlib = "*"
+matplotlib = "3.3.*"
```

Comment:
Why does the matplotlib version need to be constrained to `3.3.x`?

Reply:
Because it is apparently causing the StatisticsVolumeCorrection pipeline to crash. 
It is suggested in this link: https://stackoverflow.com/questions/67969047/error-trying-to-re-register-the-builtin-cmap-cubehelix-when-trying-to-import
I have also confirmed that the pipeline works with a version inferior to 3.4.0 (3.3.4 to be specific) !

Reply:
Are we doing this colormap registration ourselves or is it done transitively by one of our dependencies (like `yt` in the link)?

Reply:
It is done by one of our dependencies

Reply:
Could you tell me which one please? Or give me a link to a stack trace with the error 🙏

Reply:
I just sent you the relevant stack traces !

Reply:
Ok, it's `nilearn`

Reply:
Fixed since [nilearn 0.7.0](https://github.com/nilearn/nilearn/commit/0324d59a134b2fbc7c1372b2c20c117e9214ba08).

So the right solution would be to raise the minimum version of `nilearn` to 0.7.0 and leave `matplotlib` unconstrained.